tests/auth_plugin_test.php

```diff
@@ -295,6 +295,7 @@ public function test_return_correct_login_url_and_create_new_user() {
         $user->email = 'username@test.com';
         $user->firstname = 'user';
         $user->lastname = 'name';
+        $user->confirmed = 1;
```

Comment:
You don't need this line here, because  this should happen inside $this->auth->get_login_url($user) and you are testing that this happened. Please remove this line and it's good to go! Thanks for your effort.